$NetBSD: patch-doc_CMakeLists.txt,v 1.1 2013/11/15 12:14:47 rhialto Exp $

Man pages should be installed into man/, not share/man/.

--- doc/CMakeLists.txt.orig	2013-08-14 18:36:46.000000000 +0000
+++ doc/CMakeLists.txt
@@ -11,5 +11,5 @@ endif()
 
 if (NOT win32)
     file(GLOB manpages "${CMAKE_CURRENT_SOURCE_DIR}/*.1")
-    install(FILES ${manpages} DESTINATION share/man/man1)
+    install(FILES ${manpages} DESTINATION man/man1)
 endif()
